src/components/RouteLine.css

```diff
@@ -0,0 +1,24 @@
+.show-station-icon {
+    padding-top: 13%;
+}
+
+.show-station-details {
+    font-family: "Roboto","Helvetica","Arial",sans-serif;
```

Comment:
Is this different from what we see in the app. If yes dont change the font only for this